benefits/eligibility/templates/eligibility/index.html

```diff
@@ -0,0 +1,25 @@
+{% extends "core/base.html" %}
+
+{% block main_content %}
+    <div class="container">
```

Comment:
spacing should be cleaned up

Reply:
What spacing specifically? This passes djlint formatter.

Reply:
Oh wait. Or do you mean like padding.

Reply:
No I meant the code formatting... I'm not sure why, this page uses 4 as the tabsize. Our other HTML pages use 2.